### ISSUE TITLE
Missing Package

### ISSUE THREAD

**Issue by igreulich** [FormidableLabs/nodejs-dashboard#19]
When I run my app, I get the following error. (Installed the dashboard following the directions in the README from the repo.)

```
Failed to compile.

Error in ./~/nodejs-dashboard/~/pidusage/lib/stats.js
Module not found: 'child_process' in /Users/igreulich/apps/rhinofront/node_modules/nodejs-dashboard/node_modules/pidusage/lib

 @ ./~/nodejs-dashboard/~/pidusage/lib/stats.js 4:11-35 5:12-36
```

This is during the webpack build process. (I started the app with `create-react-app` but have since ejected it, am adding niceties.)

**Comment by exogen** [comment]
Hm, this seems to imply that webpack is trying to bundle a server-side only file? I can't be sure of your setup, but it's odd that webpack would touch such a file since `nodejs-dashboard` is mostly intended to run long-running server processes.

Either way, would probably be fixed by #16.

**Comment by igreulich** [comment]
I will recheck, once #16 has been addressed.

Thanks.
